### PR TITLE
[expo-file-system] fixed issues with UploadTask runtime crash and incorrect event

### PR DESCRIPTION
**Pull request by lukebrandonfarrell**
# Why

When using `createUploadTask` there would be a runtime crash due to an incorrect invocation of `toUpperCase`. Furthermore, the upload task progress event on iOS was sending the wrong value for `totalByteSent`.

- Runtime crash because .toUpperCase wasn't being invoked as a function (missing ())
- totalByteSent in upload progress callback was being invoked with `bytesSent`

# Checklist

- [X] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [X] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [X] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: packages/expo-file-system/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 ### 🐛 Bug fixes
 
+- Runtime crash due to `.toUpperCase` not being invoked as a function, it was missing `()` ([#15615](https://github.com/expo/expo/pull/15615) by [@lukebrandonfarrell](https://github.com/lukebrandonfarrell))
```
Comment (bbarthec) [inline packages/expo-file-system/CHANGELOG.md:11]:
```suggestion
- Fixed runtime crash due to `.toUpperCase` not being invoked as a function, it was missing `()`. ([#15615](https://github.com/expo/expo/pull/15615) by [@lukebrandonfarrell](https://github.com/lukebrandonfarrell))
```

File: packages/expo-file-system/CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,8 @@
...
+- `totalByteSent` in upload progress callback was incorrectly sending `bytesSent` on iOS ([#15615](https://github.com/expo/expo/pull/15615) by [@lukebrandonfarrell](https://github.com/lukebrandonfarrell))
```
Comment (bbarthec) [inline packages/expo-file-system/CHANGELOG.md:12]:
```suggestion
- Fixed `totalByteSent` in upload progress callback incorrectly sending `bytesSent` on iOS. ([#15615](https://github.com/expo/expo/pull/15615) by [@lukebrandonfarrell](https://github.com/lukebrandonfarrell))
```

**Review by bbarthec — APPROVED** [review]
It's looking good 👍 
Thank you for these two fixes 👏 
See my suggestions and we're ready to merge 👀

**Comment by lukebrandonfarrell** [comment]
Suggestions updated! @bbarthec one thing, could you let me know how we could install these changes once merged? I've not found much on updating an individual expo package from Github URL, I'm guessing you will bump to NPM?

**Comment by bbarthec** [comment]
@lukebrandonfarrell, I've just published this fix in [`expo-file-system@13.2.0`](https://www.npmjs.com/package/expo-file-system/v/13.2.0?activeTab=versions) to `npm` 🚀

**Comment by lukebrandonfarrell** [comment]
@bbarthec Just out of curiosity, I haven't seen any documentation for this API, is this undocumented? Are we open to PRs to document it? Not sure if it was deprecated intentionally

**Comment by bbarthec** [comment]
@lukebrandonfarrell, it seems this is not documented, but the docs are simply missing 🤷 and yes, we are open for any PR that might improve the codebase, including docs 😁

**Pull request merged**